### PR TITLE
Steam Age Revolution Fixes

### PR DESCRIPTION
**Pull request by CrankySupertoon**

### CODE REVIEW

**Comment by CrankySupertoon** [comment]
#195

File: docs/Mods/SteamAgeRevolution/AlloyForge.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Alloy Forge
+
+The Alloy Forge is a machine that converts two input fluids into one output fluid.
+
+## Package
+`mods.steamagerevolution.AlloyForge`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquids/IItemStack/) output** The result of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/AlloyForge.md]:
```suggestion
- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) output** The result of the recipe.
```

File: docs/Mods/SteamAgeRevolution/Crucible.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Crucible
+
+The Crucible is a machine that converts an input item into an output fluid using steam.
+
+## Package
+`mods.steamagerevolution.Crucible`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) output** The result of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Crucible.md]:
```suggestion
- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) output** The result of the recipe.
```

File: docs/Mods/SteamAgeRevolution/Distiller.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Distiller
+
+The Distiller is a machine that converts an input fluid into an output fluid and item using steam.
+
+## Package
+`mods.steamagerevolution.Distiller`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) output** The result of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Distiller.md]:
```suggestion
- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) output** The result of the recipe.
```

File: docs/Mods/SteamAgeRevolution/Distiller.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+The Distiller is a machine that converts an input fluid into an output fluid and item using steam.
+
+## Package
+`mods.steamagerevolution.Distiller`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) output** The result of the recipe.
+- **[IItemStack](/Vanilla/Items/IItemStack/) output** The result of the recipe.
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) input** The input of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Distiller.md]:
```suggestion
- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) input** The input of the recipe.
```

File: docs/Mods/SteamAgeRevolution/Vat.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Vat
+
+The Vat is a machine that converts input items and fluids into an output fluid.
+
+## Package
+`mods.steamagerevolution.Vat`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) output** The result of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Vat.md]:
```suggestion
- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) output** The result of the recipe.
```

File: docs/Mods/SteamAgeRevolution/Vat.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+The Vat is a machine that converts input items and fluids into an output fluid.
+
+## Package
+`mods.steamagerevolution.Vat`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) output** The result of the recipe.
+- **[ILiquidStack](/Vanilla/Liquid/ILiquidStack/) input** Fluid input of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Vat.md]:
```suggestion
- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) input** Fluid input of the recipe.
```

**Comment by CrankySupertoon** [comment]
I’ll squash it if you wish but I think GitHub implemented it easier

File: docs/Mods/SteamAgeRevolution/Vat.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+The Vat is a machine that converts input items and fluids into an output fluid.
+
+## Package
+`mods.steamagerevolution.Vat`
+
+## Methods
+
+- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) output** The result of the recipe.
+- **[ILiquidStack](/Vanilla/Liquids/ILiquidStack/) input** Fluid input of the recipe.
+- **[IIngredient](/Vanilla/Variable_Types/IIngredient/) inputItema** Items to input of the recipe.
```
Comment (kindlich) [inline docs/Mods/SteamAgeRevolution/Vat.md]: typo probably?
`itema` ?

Comment (CrankySupertoon) [reply docs/Mods/SteamAgeRevolution/Vat.md]: It so let me fix

**Comment by CrankySupertoon** [comment]
No clue on ticks or second I’ll look in code

**Comment by CrankySupertoon** [comment]
From me skimming the mods code it seems ticks since some machines do it for time craft 200 and that seems bad for seconds

**Comment by CrankySupertoon** [comment]
Feel free to use squash and merge

**Pull request merged**